hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/erasurecode/CodecUtil.java

```diff
@@ -78,6 +78,10 @@ public final class CodecUtil {
   public static final String IO_ERASURECODE_CODEC_XOR_RAWCODERS_KEY =
       IO_ERASURECODE_CODEC + "xor.rawcoders";
 
+  public static final String IO_ERASURECODE_CODEC_NATIVE_ENABLED_KEY = "io.erasurecode.codec.native.enabled";
```

Comment:
Hi @YuanbenWang , please fix the checkstyle. The other changes look good to me. Thanks.

Reply:
@tomscut Thank you for your suggestion. I'v updated the code and could you review again please?

Reply:
> > @tomscut Thank you for your suggestion. I'v updated the code and could you review again please?
> 
> Thanks. Now, it looks good to me.

Thank you for the review.